src/styles/blocks/main.scss

```diff
@@ -0,0 +1,32 @@
+.catalog {
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, 200px);
+  gap: 46px 48px;
+  justify-content: center;
+  justify-items: center;
+}
+
+@media (max-width: 487px) {
+  .catalog {
+    grid-template-columns: repeat(1, 200px);
+  }
+}
```

Comment:
There's a small issue with your media queries. You are using `max-width: 487px` and `min-width: 488px`. This leaves a 1px gap where neither of the styles apply. To fix this, you should use `max-width: 487.98px` for the first media query.